devito/types.py

```diff
@@ -345,6 +361,22 @@ def indexify(self, indices=None):
         indices = [a.subs(subs) for a in self.args]
         return Indexed(self.indexed, *indices)
 
+    def argument_defaults(self):
```

Comment:
if in ``Basic``, as I wrote above, we can drop some redundant code

Reply:
:+1: